### PR TITLE
fix: fix license classifier

### PR DESCRIPTION
**Pull request by joostlek**
Hey 👋🏻,

I am currently looking into licensing at Home Assistant, and I found that we could not detect the license of this library properly. According to pyproject.toml documentation about the `license` field:
> If you are using a standard, well-known license, it is not necessary to use this field. Instead, you should use one of the [classifiers](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#classifiers) starting with License ::. (As a general rule, it is a good idea to use a standard, well-known license, both to avoid confusion and because some organizations avoid software whose license is unapproved.)

If you could do a release after this PR, and maybe bump in Home Assistant, that would be awesome :)

### CODE REVIEW

**Review by bdraco — APPROVED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -6,7 +6,6 @@ authors = [
     { name = "Aarni Koskela", email = "akx@iki.fi" },
 ]
 requires-python = ">=3.9"
-license = "MIT"
```
Comment (akx) [inline pyproject.toml:9]: Why was this removed, though? I'm pretty sure there are plenty of tools that read this field rather than classifiers?

Comment (joostlek) [reply pyproject.toml:9]: Because the classifier sets it on the license field in pypi iirc

Comment (akx) [reply pyproject.toml:9]: Ah, apparently it does the thing alright:

```
(ruuvitag-ble) ~/b/ruuvitag-ble (main) $ python
>>> import importlib.metadata as im
>>> im.metadata("ruuvitag-ble")
>>> m["License-Expression"]
'MIT'
>>>
```

Never mind! :)

Comment (cdce8p) [reply pyproject.toml:9]: This PR should probably be reverted.`Hatchling` prefers the `license` field of classifier. PyPI will add support for `License-Expression` soon as well.

https://hatch.pypa.io/1.11/config/metadata/#spdx-expression
https://github.com/pypi/warehouse/issues/16620

Comment (cdce8p) [reply pyproject.toml:9]: Opened #25
